include/seqan3/io/alignment_file/format_sam.hpp

```diff
@@ -1033,11 +1035,6 @@ class alignment_file_output_format<format_sam>
                       "The id object must be a std::ranges::ForwardRange over "
                       "letters that model seqan3::Alphabet.");
 
-        static_assert((std::ranges::ForwardRange<ref_seq_type>    &&
```

Comment:
Why is the assert gone?
If it can be `std::ignore`, then you could assert that either `decays_to_ignore_v< ref_seq_type>` or the requirements are fulfilled.

Reply:
This type is not used anywhere so it should not be restricted I think